conjure_python_client/_http/requests_client.py

```diff
@@ -81,6 +87,10 @@ def _request(self, *args, **kwargs):
 
         kwargs['timeout'] = (self._connect_timeout, self._read_timeout)
 
+        if 'headers' not in kwargs:
+            kwargs['headers'] = {}
+        kwargs['headers']['X-B3-TraceId'] = fresh_trace_id()
```

Comment:
can we make the header name a constant

Reply:
done! (though ultimately shied away from using experimental mypy `Final` typing for it)